### ISSUE TITLE
Error parsing bundle asset

### ISSUE THREAD

**Issue by kagawagao** [webpack-contrib/webpack-bundle-analyzer#183]
#### Issue description
report error:
```
Error parsing bundle asset "xxx/dist/5.e2fb06a758ee3befa705.js": Cannot read property 'name' of undefined
Error parsing bundle asset "xxx/dist/13.3c2ac90a2079085d8b02.js": Cannot read property 'name' of undefined
```

#### Technical info
* Webpack Bundle Analyzer version: `2.13.0`
* Webpack version: `4.8.1`
* Node.js version: `10.1.0`
* npm/yarn version:  `6.0.1`
* OS:  `macOS High Sierra 10.13.4 (17E202)`

#### Debug info
How do you use this module? As CLI utility or as plugin?

As plugin

If CLI, what command was used? (e.g. `webpack-bundle-analyzer -O path/to/stats.json`)

If plugin, what options were provided? (e.g. `new BundleAnalyzerPlugin({ analyzerMode: 'disabled', generateStatsFile: true })`)

```js
new BundleAnalyzerPlugin({
  generateStatsFile: true
})
```

What other Webpack plugins were used?

- `webpack.DefinePlugin`
- `MomentLocalesPlugin`
- `FaviconsWebpackPlugin`
- `HtmlWebpackPlugin`
- `CopyWebpackPlugin`
- `webpack.LoaderOptionsPlugin`
- `FriendlyErrorsPlugin`
- `MiniCSSExtractPlugin`

`stats` file

[stats.json.zip](https://github.com/webpack-contrib/webpack-bundle-analyzer/files/2030362/stats.json.zip)

**Comment by kagawagao** [comment]
FYI:
this error is cause by this function
https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/12b2e371b17f7f4e34ab061da392ad6e33ff190f/src/parseUtils.js#L171-L183

`callee.object.left.object` is `undefined`

**Comment by th0r** [comment]
Could you provide one of those chunks that cause parse error?

**Comment by kagawagao** [comment]
I create a [gist](https://gist.github.com/kagawagao/71e8529f13603e22ac7f73bfca8ca6a2)

**Comment by th0r** [comment]
Fixed in `v2.13.1`
